rest/config_test.go

```diff
@@ -917,10 +918,11 @@ func TestValidateServerContext(t *testing.T) {
 	}
 
 	sharedBucketErrors := validateServerContext(sc)
-	SharedBucketError, ok := sharedBucketErrors[0].(*SharedBucketError)
-	require.True(t, ok)
-	assert.Equal(t, tb1.BucketSpec.BucketName, SharedBucketError.GetSharedBucket().bucketName)
-	assert.Subset(t, []string{"db1", "db2"}, SharedBucketError.GetSharedBucket().dbNames)
+	require.NotNil(t, sharedBucketErrors)
+	var sharedBucketError *SharedBucketError
+	require.True(t, errors.As(sharedBucketErrors.Errors[0], &sharedBucketError))
```

Comment:
length check first

Reply:
Sure, this is addressed in latest revision.